xclip.c

```diff
@@ -407,6 +407,7 @@ doIn(Window win, const char *progname)
 	    }
 	    sel_len += fread(sel_buf + sel_len, sizeof(char), sel_all - sel_len, fil_handle);
 	}
+	fclose(fil_handle);
```

Comment:
Is it possible for fil_handle to be stdin at this point. If so, are we sure it is safe to close stdin at this point? (For example, does it ever get closed twice? Are pipelines happy with their stdout (our stdin) being closed? Etc.)

Also, consider setting fil_handle = NULL after closing just to make sure it doesn't accidentally get used later.

Reply:
Thanks for the review.
I agree that it is a bad idea to fclose stdin.
I will revise my code and update the PR soon.

Reply:
Thanks for the update. Have we ruled out the possibility that this code will do anything funny like fclosing stdin twice?

Reply:
@hackerb9 I have added a check in the latest PR.